packages/vue/.eslintrc.cjs

```diff
@@ -24,6 +24,8 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     '@typescript-eslint/ban-ts-comment': 'off',
     'vue/script-setup-uses-vars': 'error',
+    // we intentionally use non-null assertion where types are inaccurate.
+    '@typescript-eslint/no-non-null-assertion': 'off',
```

Comment:
FYI, other packages don't have this setting on and use non-null assertions where appropriate.

We should align vue eslint settings with the monorepo one. Will create a ticket for that.

Reply:
So this one is coming from vue's default eslint specifically?

Reply:
Yes, specifically `@vue/eslint-config-typescript/recommended`.